config/env/internal-external.json

```diff
@@ -1,20 +1,21 @@
 {
```

Comment:
I've been doing all my testing using `config/deployment/dev-unstable.json`. If we're moving towards these new `env` based config files, can we delete the `deployment` one to avoid confusion?

Reply:
@stbrody I used env because I'm using NODE_ENV to determine what config should be used. I don't know how the deployment one works but if it's not needed then yeah I can just remove it